src/modules/launcher/Plugins/Microsoft.Plugin.Program/Programs/Win32.cs

```diff
@@ -439,16 +444,16 @@ private static Win32 ExeProgram(string path)
             try
             {
                 var program = Win32Program(path);
-                var info = FileVersionInfo.GetVersionInfo(path);
+                var info = fileVersionInfoWrapper.GetVersionInfo(path);
 
-                if (!string.IsNullOrEmpty(info.FileDescription))
+                if (!string.IsNullOrEmpty(info?.FileDescription))
                 {
                     program.Description = info.FileDescription;
                 }
 
                 return program;
             }
-            catch (Exception e) when (e is SecurityException || e is UnauthorizedAccessException)
```

Comment:
@alekhyareddy28 Removing this could hide exceptions that we are aren't expecting.  Why did we need to remove this.

Reply:
Thanks for pointing it out Ryan. I had removed it before I added a wrapper for System Operations and I had forgotten to add that back. Reverted it to how it was.